public/index.html

```diff
@@ -57,7 +57,7 @@
     <!-- Open Graph / Facebook -->
     <meta property="og:type" content="website" />
     <meta property="og:url" content="https://react-native-elements.js.org/" />
-    <meta property="og:title" content="Playground 🚀 | React Native Elements" />
+    <meta property="og:title" content="Playground | React Native Elements" />
```

Comment:
??

Reply:
Hey Pranshu , I will undo this right away.